src/methods.js

```diff
@@ -24,7 +24,9 @@ var methods = {};
             verdictLevel,
             score;
 
-        if (options === undefined) { return; }
+        if (typeof(options) === "undefined") {
```

Comment:
Have pushed a change back to original, but including single-line -> multi-line change.